config/application.yml

```diff
@@ -1,8 +1,8 @@
 default: &default
-  DB_NAME: "ic_rails_huey_toby"
+  DB_NAME: "postgres"
   DB_HOST: "localhost"
   DB_PORT: "5432"
-  DB_USERNAME: "postgres"
+  DB_USERNAME: "tuubz"
```

Comment:
You need to change your local setup to use the default `postgres` username 🙈

Reply:
Hahah yeah, I need to make work of it 🙈